.travis.yml

```diff
@@ -1,58 +1,51 @@
 language: python
-python: 2.7
 sudo: false
+
 cache:
   pip: true
   directories:
     - $HOME/buildout-cache
+
 before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - mkdir $HOME/.buildout
-  - echo "[buildout]" > $HOME/.buildout/default.cfg
-  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
-  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install --upgrade pip setuptools zc.buildout
-  - if [[ `npm -v` != 3.10.7 ]]; then npm i -g npm@3.10.7; fi
-  - npm install -g grunt-cli
-
-env:
-  matrix:
```

Comment:
@tisto ok, please have a look again at the file https://github.com/plone/bobtemplates.plone/blob/tox/tests/test_skeleton_generation.py I have changed that so that it reflects your test matrix.